src/portal/azext_portal/generated/_validators.py

```diff
@@ -0,0 +1,18 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+def example_name_or_id_validator(cmd, namespace):
```

Comment:
Maybe you should delete it.

Reply:
This is auto-generated, can you explain more about why it should be deleted?